### PR TITLE
Adding myself

### PR DESCRIPTION
**Pull request by glennblock**

### CODE REVIEW

File: INTERESTED_DEVELOPERS.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ If you want to be listed here, open a PR with your information, just order yours
 * @sungam3r
   * Company/Project/Repo: https://github.com/graphql-dotnet/server
   * Reason: Interested in client/server spec
+* @glennblock
+  * Company/Project/Report: https://github.com/microsoft
+  * Reason: Interested in a common HTTP spec, and in adoption of emerging HTTP standards like HTTP SEARCH
```
Comment (benjie) [inline INTERESTED_DEVELOPERS.md:60]: Please move this up so it's in alphabetical order by username :+1:

**Review by benjie — CHANGES_REQUESTED** [review]
Welcome!

**Comment by IvanGoncharov** [comment]
@glennblock Welcome @glennblock 👋

File: INTERESTED_DEVELOPERS.md (hunk 1 above)
Comment (glennblock) [reply INTERESTED_DEVELOPERS.md:60]: Hi @benjie, I will fix and send a new PR

**Pull request merged**